packages/k8s/src/hooks/prepare-job.ts

```diff
@@ -166,14 +167,17 @@ function createPodSpec(
   const podContainer = {
     name,
     image: container.image,
-    command: [container.entryPoint],
     args: container.entryPointArgs,
     ports: containerPorts(container)
   } as k8s.V1Container
   if (container.workingDirectory) {
     podContainer.workingDir = container.workingDirectory
   }
 
+  if (container.entryPoint) {
+    podContainer.command = [container.entryPoint]
+  }
+
```

Comment:
We should avoid setting up args from container and default to the same check like for the entrypoint:
```js
if (container.entryPointArgs?.length) {
    podContainer.args = container.entryPointArgs
}
```

Reply:
Good point!